addon/bootstrap.js

```diff
@@ -20,6 +20,9 @@ Cu.import("resource://gre/modules/Log.jsm");
 const STUDY_UTILS_PATH = `${__SCRIPT_URI_SPEC__}/../${studyConfig.studyUtilsPath}`;
 const HOMEPAGE_STUDY_PATH = `${__SCRIPT_URI_SPEC__}/../HomepageStudy.jsm`;
 
+const SIMULATOR_PATH = `${__SCRIPT_URI_SPEC__}/../Simulator.jsm`;
```

Comment:
nit: move this on the previous line (22) and just leave one blank line after, not 2

Reply:
Done